phylogenetic/build-configs/ci/config.yaml

```diff
@@ -1,2 +1,9 @@
-custom_rules:
-  - build-configs/ci/copy_example_data.smk
+inputs:
+  - name: ncbi
+    metadata: "example_data/metadata.tsv"
+    sequences: "example_data/sequences.fasta"
+
+additional_inputs:
+  - name: usvi
+    metadata: "example_data/metadata_usvi.tsv"
+    sequences: "example_data/sequences_usvi.fasta"
```

Comment:
Let's use the same pattern here as the default workflow and specify both ncbi & usvi as entries in `inputs` rather than `additional_inputs`. (I know we don't use overlay's with CI so your approach isn't a problem, but I think it's nicer to be consistent.)

Reply:
Fixed with https://github.com/nextstrain/zika/pull/80/commits/69eff794b8f59f0df9bb1264b1c747acf2c9116a